resources/definitions/fdmprinter.def.json

```diff
@@ -2013,6 +2013,29 @@
                     "value": "jerk_layer_0",
                     "enabled": "jerk_enabled",
                     "settable_per_mesh": false
+                },
```

Comment:
These new settings should be set just below speed_slowdown_layers, not below all acceleration and jerk settings.